### DISCUSSION TITLE
Can't run custom script

### DISCUSSION THREAD

**Discussion by DrSensor** [lumeland/lume#346] [Q&A] [answered]
```js
// _config.ts
import lume from "lume/mod.ts";

const site = lume();

console.info("Deno.args:", Deno.args);
site.script("hello", () => console.log("hello world"));

export default site;
```
```console
$ deno task lume run hello
Task lume echo "import 'lume/cli.ts'" | deno run --unstable -A - "run" "hello"
Loading config file file:///run/media/wildan/Projects/Examples/lume-example/_config.ts

Deno.args: [ "run", "hello" ]

⚡️ h
/bin/bash: line 1: h: command not found
```
🤔 Is it just me?
Or does `lume run <script>` (mentioned in [the docs](https://lume.land/docs/core/scripts/) without using `deno task`) is in progress for deprecation?

**Comment by oscarotero** [comment, marked as answer]
Yes, it was a bug.
At some point, cliffy (the library I use to build the CLI) changed the way the variadic arguments are passed. I just fixed it in https://github.com/lumeland/lume/commit/2c59a27c4f7c595cf3aced4b31e52e3b657a3a0a

I cannot release a new version right now because i'm in the middle of something, but you can use the development version (`lume upgrade --dev`).

**Reply by oscarotero** [reply]
This is fixed in Lume v1.15.0
